library/src/main/java/com/novoda/downloadmanager/lib/DownloadManager.java

```diff
@@ -646,7 +645,8 @@ public Cursor query(Query query) {
         if (underlyingCursor == null) {
             return null;
         }
-        return new CursorTranslator(underlyingCursor, downloadsUriProvider.getDownloadsByBatchUri());
+        StatusTranslator statusTranslator = new StatusTranslator();
```

Comment:
OK I'm not bothered too much :) just want to avoid allocating on our hot paths, but I'm sure this is a small optimisation in comparision to others :laughing: